Tests/Kernel/TestMemoryDeviceMmap.cpp

```diff
@@ -34,18 +89,18 @@ TEST_CASE(test_memory_access_device_mmap)
     // however, it will be much nicer to have some sort of a node in the ProcFS
     // to expose physical memory ranges (e820 memory map).
 
-    auto result = mem_chunk(fd, 0xe0000, 0x100000 - 0xe0000);
-    EXPECT_EQ(result, true);
+    auto mmap_result = mem_chunk(fd, 0xe0000, 0x100000 - 0xe0000);
```

Comment:
Not related to your changes, could be a future PR, but shouldn't we `munmap` these?

Reply:
We probably should. it will be fixed later in a future PR :)

Reply:
Being fixed in #11895.